contributing.md

```diff
@@ -10,21 +10,43 @@ You can always add to an API, you cannot ever remove anything from one. If the d
+
+
+## <a name="question"></a> Questions
+Please do not open issues for general support questions and keep our GitHub issues for bug reports and feature requests. There is a much better chance of getting your question answered on [StackOverflow](https://stackoverflow.com/search?q=uwp+community+toolkit) where questions should be tagged with the tag `uwp-community-toolkit`
+
+## <a name="issue"></a> Found a Bug?
+If you find a bug, you can help us by
+[submitting an issue](https://github.com/Microsoft/UWPCommunityToolkit/issues). Even better, you can
+[submit a Pull Request](#pr) with a fix.
+
+## <a name="pr"></a> Submitting a pull request
+For every controbution, you must:
+
+* test your code with the [supported SDKs](readme.md#supported)
+* follow the [quality guidance](#xaml), [general rules](#rules) and [naming convention](#naming)
+* target master branch (or an appropriate rel branch if appropriate for a bug fix)
+
+* If adding a new feature
+    * Before starting coding, **you should open an uservoice entry** under "UWP Community Toolkit" on [Uservoice](https://wpdev.uservoice.com/forums/110705-universal-windows-platform/category/193402-uwp-community-toolkit) and start discussing with the community to see if your idea/feature is interesting enough. 
+    * Add or update a documentation page in the [documentation folder](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/docs). Once validated your documentation will be visible [here](http://docs.uwpcommunitytoolkit.com/en/master/)
+    * Add or update a sample for the [Sample app](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp)
+    * Add or update unit tests (if applicable)
 
-* Before starting coding, **you should open an uservoice entry** under "UWP Community Toolkit" on [Uservoice](https://wpdev.uservoice.com/forums/110705-universal-windows-platform/category/193402-uwp-community-toolkit) and start discussing with the community to see if your idea/feature is interesting enough. 
-* A documentation page in the [documentation folder](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/docs). Once validated your documentation will be visible [here](http://docs.uwpcommunitytoolkit.com/en/master/)
-* A sample for the [Sample app](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp) (If applicable)
-* Unit tests (If applicable)
-* You tested your code with SDK 14393 and SDK 15063
-* PR has to target master branch
 
 PR has to be validated by at least two core members before being merged.
 
 Once merged, you can get a pre-release package of the toolkit by adding this ([Nuget repo](https://dotnet.myget.org/F/uwpcommunitytoolkit/api/v3/index.json) | [Gallery](https://dotnet.myget.org/gallery/uwpcommunitytoolkit)) to your Visual Studio.
 
-## Quality insurance for pull requests for XAML controls
+## <a name="xaml"></a> Quality insurance for pull requests for XAML controls
```

Comment:
I don't think we need `insurance`, but rather `assurance`

Reply:
Good catch, it's been like that for a year :)